docs/setup/production.asciidoc

```diff
@@ -109,6 +109,35 @@ transport.tcp.port: 9300 - 9400
 elasticsearch.hosts: ["http://localhost:9200"]
 --------
 
+=== Load Balancing Across Multiple Kibana Instances
```

Comment:
Right now, this content appears in its own page. To include this content on the production page, include [float] above the header name.  Also, add a link to this content in the bulleted list at the top of page.

We use sentence caps for titles:  Load balancing across multiple Kibana instances

Reply:
👍 ! de4f19b